src/Extensions/Activities/Internal/EventSource/ActivityEventSender.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace Microsoft.Omex.Extensions.Activities
 {
-	internal sealed class ActivityEventSender : IActivitiesEventSender
+	internal sealed class ActivityEventSender : IActivitiesEventSender, IActivityStopObserver
```

Comment:
Why use the interface if it is not implemented?

Reply:
Sorry this part committed by mistakes